src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion_inpaint.py

```diff
@@ -559,7 +559,7 @@ def prepare_extra_step_kwargs(self, generator, eta):
         return extra_step_kwargs
 
     # Copied from diffusers.pipelines.stable_diffusion.pipeline_stable_diffusion.StableDiffusionPipeline.decode_latents
-    def decode_latents(self, latents):
+    def decode_latents(self, latents, image=None, mask=None):
```

Comment:
we we add `image` and `mask` here but did not use them?

Reply:
we pass them if vae is `AsymmetricAutoencoderKL`

Reply:
so these two arguments (`image` and `mask`) are not used here at all - something is wrong here 

maybe this is what you need? 
```python
        latents = 1 / self.vae.config.scaling_factor * latents
        if isinstance(self.vae, AsymmetricAutoencoderKL):
                image = self.vae.decode(latents, image, mask, return_dict=False)[0]
        else:
                 image = self.vae.decode(latents, return_dict=False)[0]
        image = (image / 2 + 0.5).clamp(0, 1)
        image = image.cpu().permute(0, 2, 3, 1).float().numpy()
```

Reply:
there is a dict called `condition_kwargs`
if vae is `AsymmetricAutoencoderKL`, image and mask are added to the dict and passed to the decode method
otherwise dict remains empty

Reply:
I just realize this method `decode_latents` is not used at all 
let's delete?

Reply:
oh, sure